frontend/src/api/services/answer_survey.test.ts

```diff
@@ -0,0 +1,80 @@
+import { AxiosInstance, AxiosResponse } from 'axios';
+import { httpClient } from '../client';
+import { AnswerSurvey, AnswerSurveyCreate } from '@api/models/answer_survey';
+import { AnswerSurveyService } from './answer_survey';
+
+// Mocking httpClient
+jest.mock('../client', () => ({
+  httpClient: {
+    post: jest.fn()
+  }
+}));
+
+// expected result to tests
+const mockResponse = {
+  mock: true
+};
+
+// Mock survey create
+const mockAnswerSurveyCreate: AnswerSurveyCreate = {
+  survey_id: 1,
+};
+
+describe('AnswerSurveyService', () => {
+  let instance: AnswerSurveyService;
+  let res: AxiosResponse<AnswerSurvey>;
+
+  describe('register method', () => {
+    describe('when returns the expected data', () => {
```

Comment:
@mizek1 , this is just a bikeshedding / common mistake among Portuguese speakers, @amiapmorais and I do the same.
```suggestion
    describe('when returns expected data', () => {
```

Reply:
I too..

Reply:
Yeah! This error is in other test files on the system (which I was inspired to create these).

Reply:
This happens a lot, usually I also make it so hope the guys find out lol